### PR TITLE
Flake8 fixes pt3

### PR DESCRIPTION
**Pull request by cadenmyers13**

### CODE REVIEW

**Comment by sbillinge** [comment]
@cadenmyers13 it's a drag I am afraid but every file we touch we should update the copyright statement at the top to give a date range ending in 2024 for the trustees of CU.

File: src/diffpy/pdfgui/gui/phaseconfigurepanel.py [hunk 1]
```diff
@@ -319,7 +319,7 @@ def restrictConstrainedParameters(self):
         """Set 'read-only' boxes that correspond to constrained parameters."""
 
         self.setToolTips(tooltips.phasepanel)
-        txtbg = self.textCtrlA.DefaultStyle.BackgroundColour
+        # txtbg = self.textCtrlA.DefaultStyle.BackgroundColour
```
Comment (sbillinge) [inline src/diffpy/pdfgui/gui/phaseconfigurepanel.py:322]: This change will change behavior. If flake8 is complaining about txtbg not being used, just remove the `txtbg =` from the beginning of the line.

File: src/diffpy/pdfgui/gui/temperatureseriespanel.py [hunk 2]
```diff
@@ -293,13 +301,19 @@ def onColClick(self, event):  # wxGlade: TemperatureSeriesPanel.<event_handler>
         column = event.GetColumn()
         # sort by temperature
         if column == 0:
-            sortkey = lambda tf: float(tf[0])
+
+            def sortkey(tf):
```
Comment (sbillinge) [inline src/diffpy/pdfgui/gui/temperatureseriespanel.py:305]: Define the function elsewhere then use it here

File: src/diffpy/pdfgui/gui/temperatureseriespanel.py [hunk 3]
```diff
@@ -293,13 +301,19 @@ def onColClick(self, event):  # wxGlade: TemperatureSeriesPanel.<event_handler>
...
+
         # sort by filename with numerical comparison of digits
         elif column == 1:
             filenames = [f for t, f in self.datasets]
             numericStringSort(filenames)
             order = dict(zip(filenames, range(len(filenames))))
-            sortkey = lambda tf: order[tf[1]]
+
+            def sortkey(tf):
+                return order[tf[1]]
```
Comment (sbillinge) [inline src/diffpy/pdfgui/gui/temperatureseriespanel.py:315]: See above

**Review by sbillinge — COMMENTED** [review]
A few more comments on this one

**Comment by cadenmyers13** [comment]
> @cadenmyers13 it's a drag I am afraid but every file we touch we should update the copyright statement at the top to give a date range ending in 2024 for the trustees of CU.

I see in your edit you had it as trustees of michigan state. Do we want to do both or just Columbia?

**Pull request closed without merging**